lib/routes/swagger.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const swaggerUI = require('swagger-ui-express');
+const swaggerDefs = require('../../lib/swagger.json');
```

Comment:
is there a reason this can't be `../swagger.json`?

Reply:
.... no. no there's not. I don't even. what? how?
![what?](https://media.giphy.com/media/nV9i4o1cf1atG/giphy.gif)